src/schema/schema_create.c

```diff
@@ -248,12 +257,14 @@ __create_file(WT_SESSION_IMPL *session, const char *uri, bool exclusive, const c
                  * To prevent mistakes with API usage, we should return an error here rather than
                  * inferring a repair.
                  */
+                WT_STAT_CONN_INCR(session, session_table_create_with_import_fail);
                 WT_ERR_MSG(session, EINVAL,
                   "%s: import requires that 'file_metadata' or 'metadata_file' is specified or the "
                   "'repair' option is provided",
                   uri);
             }
         }
+        WT_STAT_CONN_INCR(session, session_table_create_with_import_success);
```

Comment:
@sueloverso are we interested in statistics of imported files, tables, or both? We make one call to import a table and that call imports multiple files. The names `session_table_create_with_import_fail` and `session_table_create_with_import_success` are more related to tables not files.

Reply:
These statistics are generally at the API level. So it is a count of calls that involved import. I think that is sufficient.